atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/OracleDbKvsEteTestSuite.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.ete;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.atlasdb.ete.coordination.CoordinationEteTest;
+import java.time.Duration;
+import org.junit.ClassRule;
+import org.junit.rules.RuleChain;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    TodoEteTest.class,
+    TimestampManagementEteTest.class,
+    CoordinationEteTest.class,
+    LockWithoutTimelockEteTest.class
+})
+public class OracleDbKvsEteTestSuite extends EteSetup {
```

Comment:
Why do we not run `LockWatchEteTest`? Only asking because we seem to run those on Postgres

Reply:
Actually, I remember why now: the LW one is used only where you have a timelock-server (which is required for LW). I was having difficulty getting Oracle-backed DB-timelock to run properly here (too many containers, perhaps)? so ended up not doing that. That Oracle-backed DB-timelock kind of works is tested internally, so chose not to replicate it here.

Reply:
I see, this is fine in that case